doc/icicle-concrete.md

```diff
@@ -498,6 +491,12 @@ if (feat != feat_end) {
 return days;
 ```
 
+Datalog
+```
+;; Basically a the min of the max 5
```

Comment:
Might have misunderstood the example, isn't it asking for the nth day since the Earliest, where n is 5 here?